diff --git a/src/core/manifest.js b/src/core/manifest.js
index fa5af51c..a2eb1b7e 100644
--- a/src/core/manifest.js
+++ b/src/core/manifest.js
@@ -54,7 +54,7 @@ Object.defineProperty(object, 'old_mirrors', {
  */
 Object.defineProperty(object, 'cub_mirrors', { 
     get: ()=> {
-        let lampa = ['cub.best', 'cub.black', 'durex.monster', 'cubnotrip.top']
+        let lampa = ['{{CUB_DOMAIN}}']
         let users = localStorage.getItem('cub_mirrors') || '[]'
 
         try {
